README.md

```diff
@@ -12,15 +12,20 @@ A Quick-start Guide for people that want to **D**o **W**hat **Y**ou **L**ove!
 The world is not perfect.  
 We have all worked somewhere we _didn't **love**_. <br />
 We have _all_ felt the _frustration_ of using products that have flaws
-(and been unable to fix them).   
-**The aim of _dwyl_ is to address *both* of these issues _simultaneously_.**
-
-If you'd like to know more, _please **read our [manifesto](/manifesto.md)**_.  
+that _we_ were unable to fix ourselves.   
+We have all experienced being in an environment that
+_didn't support us_ in doing all we can do in the world.
+**The aim of _dwyl_ is to address *all* of these issues _simultaneously_:**
+https://github.com/dwyl/start-here/blob/master/mission.md
+
+If you'd like to know more about what we value,
+_please **read our [manifesto](https://github.com/dwyl/start-here/blob/master/manifesto.md)**_.  
  The short version is:
 "_**Treat others as you would like others to treat you**_" ~
-[***The Golden Rule***](/manifesto.md)
+[***The Golden Rule***](https://github.com/dwyl/start-here/blob/master/manifesto.md)
 
-If you want to be part of the solution, join us: http://www.dwyl.io/
+If you want to be part of the solution, join us:
+https://github.com/dwyl/start-here/issues/143
```

Comment:
Not a fan of including this issue/thread in the `README.md` because this is _not_ a good way of joining a community. Being forced to scroll through 97 comments from random people is not a great "onboarding" experience.
![image](https://user-images.githubusercontent.com/194400/73192498-06153200-4121-11ea-997a-c3dc30829e06.png)

I would _much_ rather focus a _Sprint_ of effort into "Onboarding" people.
Such that they get a "tour" of both our app, learning resources and community.
(_obvs focussing on the App for now because that's what we **need** to focus on_!)